src/app.js

```diff
@@ -1,12 +1,70 @@
-import axios from 'axios';
+import axios from "axios";
 
-// Async - await Full data
-async function fetchData() {
+const countryList = document.getElementById('country-list');
+// const countryImage = document.getElementById('country-flag')
+
+//1. data fetchen
+
+async function fetchDataCountries() {
+    const URI = 'https://restcountries.com/v2/all'
+    // const ENDPOINT = 'name'
     try {
-        const result = await axios.get('https://restcountries.com/v2/all');
-        console.log(result)
-    } catch(error) {
-        console.error(error);
+        const response = await axios.get(URI)
+        console.log(response.data);
+
+        response.data.sort ((a, b) => (a.population - b.population));
+
+            response.data.map((country) => {
+                const itemName = document.createElement('li');
+                const itemFlag = document.createElement('img');
+                const itemPopulation = document.createElement('li');
+                //maakt die een class element aan die je in styles kunt gebruiken
+                itemName.setAttribute('class', 'countryName');
+                itemFlag.setAttribute('class', 'countryImage');
+                itemPopulation.setAttribute('class', 'countryPopulation');
+                itemFlag.setAttribute('src', country.flag);
+                itemPopulation.textContent = "population of : " + country.population;
+                // sortPopulation(country.population)
+
+                switch(country.region) {
```

Comment:
Lijkt mij dat hier classes gebruikt moeten worden. meerdere keren dezelfde id op een pagina is volgens mij niet de bedoeling.